documentation/asciidoc/topics/ref_container_storage.adoc

```diff
@@ -1,15 +1,17 @@
 [id='storage_resources-{context}']
 = Storage Resources
 By default, {ispn_operator} allocates `1Gi` for storage for both
-{cacheservice} and {datagridservice} nodes. You can configure storage size for
-{datagridservice} nodes but not {cacheservice} nodes.
+{cacheservice} and {datagridservice} nodes. You can configure storage resources
+for {datagridservice} nodes but not {cacheservice} nodes.
 
 [source,options="nowrap",subs=attributes+]
 ----
 include::yaml/cr_container_storage.yaml[]
 ----
 
 <1> Configures the storage size for {datagridservice} nodes.
+<2> Defines whether storage is ephemeral or permanent. Set the value to `true` to use ephemeral storage, which means all data in storage is deleted when clusters shut down or restart. The default value is `false`, which means storage is permanent.
```

Comment:
Do we have ephemeral storage in 2.0.x? Isn't it only on master?

Reply:
whoops. I've removed ephemeral storage from this PR. nice catch @rigazilla